### PR TITLE
Addding auth pages

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Hook/useTokenChecker.js
- [ ] src/Pages/AuthPage/ProfileComponents/AuthContainer/AuthContainer.jsx
- [ ] src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx
- [ ] src/Pages/AuthPage/ProfileComponents/SignupBox/SignupBox.jsx
- [ ] src/context/Auth.context.js

## What not to check
- [ ] Do Not Check Any Other Files

live site,
https://stormpoint-shop.netlify.app/

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Gone through whole file and didn't found a single mistake, well structured file and clean code.
Good to merge

File: src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import { useEffect } from "react";
```
Comment (Dev-AkshatGupta) [inline src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx:3]: Can import it with useState only just a small suggestion

File: src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import { AuthContainer } from "../AuthContainer/AuthContainer";
+import { SignupBox } from "../SignupBox/SignupBox";
+
+export const LoginBox = () => {
+  const { login, testLogger } = useAuthContext();
+  const [userDetails, setUserDetails] = useState({ email: "", password: "" });
+  const [error, setError] = useState({ state: false, text: "" });
+  const [type, setType] = useState(true);
+
+  const validation = /^(?=.*\d)(?=.*[a-z]).{5,10}$/;
```
Comment (Dev-AkshatGupta) [inline src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx:14]: This is really amazing you are taking care of all the edge cases
Kudos to that man

**Review by Dev-AkshatGupta — APPROVED** [review]
you can merge it this whole thing looks really cool to me

File: src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx (hunk 2 above)
Comment (b0llu) [reply src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx:14]: thanks

File: src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx (hunk 1 above)
Comment (b0llu) [reply src/Pages/AuthPage/ProfileComponents/LoginBox/LoginBox.jsx:3]: thanks for pointing it out i will make the changes

**Pull request merged**